source/Octopus.Tentacle.Tests.Integration/ClientGathersRpcCallMetrics.cs

```diff
@@ -214,7 +214,7 @@ private static void ThenClientOperationMetricsShouldBeFailed(ClientOperationMetr
             metric.Exception.Should().BeEquivalentTo(expectedException);
             metric.WasCancelled.Should().BeFalse();
 
-            metric.End.Should().BeAfter(metric.Start);
+            metric.End.Should().BeOnOrAfter(metric.Start);
```

Comment:
It's possible this takes no time. Haven't seen this path fail, but the path above was so allowing this path to do the same.